Bio/NMR/xpktools.py

```diff
@@ -114,7 +114,7 @@ def __init__(self, infn):
             self.data = [line.split("\012")[0] for line in infile]
 
     def residue_dict(self, index):
-        """Return a dict of lines in \`data\` indexed by residue number or a nucleus.
+        r"""Return a dict of lines in \`data\` indexed by residue number or a nucleus.
```

Comment:
Rather than adding the leading r for raw mode, can you just remove the escaped back ticks? For RST I think plain backticks would be fine.